Sources/BSON/Date+BSONValue.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import NIO
+
+extension Date: BSONValue {
+    static var bsonType: BSONType { .datetime }
```

Comment:
this extension and and the one for `String` need some acls

Reply:
I obviously rely too much on the linter! thanks for catching these